### PR TITLE
First version blogpost Michaëla

### PR DESCRIPTION
**Pull request by Deryaa**

### CODE REVIEW

File: _posts/2019-04-02-The-Scrum-framework-is-a-liberating-structure.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+---
+
+<strong>Scrum is an Agile <em>framework</em>.</strong>
+
+What does that actually mean?
+Are we supposed to fill it? 
+Could we grab it, put it on the wall and paint our image in it, the way we see it, the way it works for us?
+
+Probably, we can. 
+Seemingly, many organizations attempt to connect the numbers and dots in precisely that way, trying to get a hold on their portfolio planning, and using Scrum for metrics to fill their project reports - solely focusing on accelerating output and pushing velocity through the roof.
```
Comment (KevinVHoutte) [inline _posts/2019-04-02-The-Scrum-framework-is-a-liberating-structure.md]: leave the comma
```suggestion
Seemingly, many organizations attempt to connect the numbers and dots in precisely that way, trying to get a hold on their portfolio planning and utilising Scrum for metrics to fill their project reports - solely focusing on accelerating output and pushing velocity through the roof. 
```

File: _posts/2019-04-02-The-Scrum-framework-is-a-liberating-structure.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+What does that actually mean?
+Are we supposed to fill it? 
+Could we grab it, put it on the wall and paint our image in it, the way we see it, the way it works for us?
+
+Probably, we can. 
+Seemingly, many organizations attempt to connect the numbers and dots in precisely that way, trying to get a hold on their portfolio planning, and using Scrum for metrics to fill their project reports - solely focusing on accelerating output and pushing velocity through the roof. 
+But let's be careful, since we can easily get trapped in this. 
+A framework isn't there to just fill it and use it. 
+Did we take the time, or do the effort, to ponder over further possibilities, opportunities, that the frame might entail?
```
Comment (KevinVHoutte) [inline _posts/2019-04-02-The-Scrum-framework-is-a-liberating-structure.md]:
```suggestion
Did we take the time or the effort to ponder over further possibilities that the frame might entail?
```

File: _posts/2019-04-02-The-Scrum-framework-is-a-liberating-structure.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+The power of the Scrum framework is in its simplicity. 
+The predictively recurring Scrum events form a synchronized pulse creating a steady heartbeat for the members of the Scrum team. 
+The drone is persistently present, but only discretely so, not drawing attention, because the frame is not what it's about. 
+It's a minimalist frame, that vanishes after a while, and is merely setting the scene... trying to create a fertile setting for a space in which, as soon as the Scrum team is accustomed to the pattern of the heartbeat, all team members can focus - as one organism - on collaboration, innovation and co-creation.
+
+<img alt="Together, we create" src="{{ '/img/scrum-is-a-liberating-structure/together-we-create.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 750px;">
+
+Interhuman friction due to role shifts, adapting to new responsibilities and accountabilities, as well as difficulties with alignment of expectations are well known characteristics of a team's storming phase. 
+But Tuckman, with his forming, storming, norming and performing model of team dynamics, was only partly right.
```
Comment (KevinVHoutte) [inline _posts/2019-04-02-The-Scrum-framework-is-a-liberating-structure.md]:
```suggestion
But Tuckman, with his forming, storming, norming and performing model of team dynamics was only partly right. 
```

File: _posts/2019-04-02-The-Scrum-framework-is-a-liberating-structure.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+The frame and heartbeat might appear to be limiting structures at first, but once applied as intended, prove to provide liberation – within the safety of knowing that the entire team is referencing through the same framework.
+
+Metaphorically: Imagine a team looking at the sea and the framework is not there. 
+What are we looking at? 
+What are we seeing? 
+What area of the horizon are we exploring? 
+
+It will be hard to be aligned unless someone or something tells us where to look and what to look for. 
+Then put the frame, the window, on the beach. 
+All of us are now looking through it – together exploring a far bigger wedge of the sea than we ever could if we were alone.
```
Comment (KevinVHoutte) [inline _posts/2019-04-02-The-Scrum-framework-is-a-liberating-structure.md]:
```suggestion
All of us are looking through it – together, exploring a far bigger wedge of the sea than we ever could imagine doing alone.
```

File: _posts/2019-04-02-The-Scrum-framework-is-a-liberating-structure.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+The power of the Scrum framework is in its simplicity. 
+The predictively recurring Scrum events form a synchronized pulse creating a steady heartbeat for the members of the Scrum team. 
+The drone is persistently present, but only discretely so, not drawing attention, because the frame is not what it's about. 
+It's a minimalist frame, that vanishes after a while, and is merely setting the scene... trying to create a fertile setting for a space in which, as soon as the Scrum team is accustomed to the pattern of the heartbeat, all team members can focus - as one organism - on collaboration, innovation and co-creation.
+
+<img alt="Together, we create" src="{{ '/img/scrum-is-a-liberating-structure/together-we-create.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 750px;">
+
+Interhuman friction due to role shifts, adapting to new responsibilities and accountabilities, as well as difficulties with alignment of expectations are well known characteristics of a team's storming phase.
```
Comment (yannickdeturck) [inline _posts/2019-04-02-The-Scrum-framework-is-a-liberating-structure.md]:
```suggestion
Interhuman friction due to role shifts, adapting to new responsibilities and accountabilities, as well as difficulties with alignment of expectations are well-known characteristics of a team's storming phase. 
```

**Pull request merged**